torch/onnx/_constants.py

```diff
@@ -1,9 +1,10 @@
 """Constant values used in ONNX."""
 
 ONNX_ARCHIVE_MODEL_PROTO_NAME = "__MODEL_PROTO"
-onnx_default_opset = 14
-onnx_main_opset = 17
-onnx_stable_opsets = tuple(range(7, onnx_main_opset))
```

Comment:
nit: Please add in description why `onnx_stable_opsets` are removed.

Reply:
Added